xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -86,18 +89,10 @@ static Bootstrapper getInstance() throws Exception {
     }
 
     @VisibleForTesting
-    FileBasedBootstrapper(Bootstrap bootstrapConfig) throws IOException {
-      ApiConfigSource serverConfig = bootstrapConfig.getXdsServer();
-      if (!serverConfig.getApiType().equals(ApiType.GRPC)) {
-        throw new IOException("Unexpected api type: " + serverConfig.getApiType().toString());
-      }
-      if (serverConfig.getGrpcServicesCount() != 1) {
-        throw new IOException(
-            "Unexpected number of gRPC services: expected: 1, actual: "
-                + serverConfig.getGrpcServicesCount());
-      }
-      balancerName = serverConfig.getGrpcServices(0).getGoogleGrpc().getTargetUri();
-      node = bootstrapConfig.getNode();
+    FileBasedBootstrapper(BootstrapInfo bootstrapInfo) {
+      this.balancerName = bootstrapInfo.serverConfig.uri;
```

Comment:
s/balancerName/serverUri/ ? One of the changes will be needed in the spec is "Remove balancer_name field from experimental_xds LB policy config"

Reply:
Done.